apps/nestjs-api/src/reminder-emails/reminder-emails.service.ts

```diff
@@ -0,0 +1,288 @@
+import { Injectable } from '@nestjs/common'
+import {
+  ConnectProfileStatus,
+  MentorshipMatchStatus,
+  RediLocation,
+  UserType,
+} from '@talent-connect/common-types'
+import * as AWS from 'aws-sdk'
+import * as jsforce from 'jsforce'
+import { ConMentorshipMatchesService } from '../con-mentorship-matches/con-mentorship-matches.service'
+import { ConProfilesService } from '../con-profiles/con-profiles.service'
+import { SfApiEmailTemplatesService } from '../salesforce-api/sf-api-email-templates.service'
+
+const sesConfig = {
+  accessKeyId: process.env.NX_EMAILER_AWS_ACCESS_KEY,
+  secretAccessKey: process.env.NX_EMAILER_AWS_SECRET_KEY,
+  region: process.env.NX_EMAILER_AWS_REGION,
+}
+
+const isProductionOrDemonstration = () =>
+  ['production', 'demonstration', 'staging'].includes(process.env.NODE_ENV)
+
+@Injectable()
+export class ReminderEmailsService {
+  private ses: AWS.SES
+
+  constructor(
+    private readonly emailTemplatesService: SfApiEmailTemplatesService,
+    private readonly conProfilesService: ConProfilesService,
+    private readonly conMentorshipMatchesService: ConMentorshipMatchesService
+  ) {
+    this.ses = new AWS.SES(sesConfig)
+  }
+
+  async getDraftingConProfiles({ userType }: { userType: UserType }) {
+    if (!userType) {
```

Comment:
Superfluous? The only client of this class is the controller, TS should guarantee we don't run into `userType` being falsy.